academy/4-my-own-chain/wager-denom.md

```diff
@@ -99,6 +99,33 @@ You have prepared the ground by placing this token denomination into the relevan
     )
     ```
 
+## Live testing with a relayer
+
+With the checkers application ready to accommodate IBC-foreign tokens, you should run some tests locally with another blockchain's tokens without running a large-scale operation. Starport luckily has built in the [Typescript relayer](https://docs.starport.com/kb/relayer.html). If you look at the GUI Starport created in your checkers blockchain, you will see a _Relayers_ section on the left.
+
+A relayer is a process that transfers IBC packets between two blockchains. It just so happens that here this process is **running in your browser** using the account you configured in your browser. The account is the same one you would use to play a game of checkers. Dub it `alice123@checkers`.
+
+1. On the checkers end, the relayer is already configured to connect to your running checkers blockchain and to use the tokens of whichever account you have configured in your browser, here `alice123@checkers`. Therefore, it gets the same privileges to access your tokens that you have granted to the checkers' browser application.
+2. You need to configure it to connect to the other blockchain hosting the foreign tokens you want to transfer. It can be the Cosmos Hub or a [testnet](https://tutorials.cosmos.network/connecting-to-testnet/testnet-tutorial.html) that you or someone else runs.
```

Comment:
This page has been removed (tutorials.cosmos.network is this platform)

Reply:
@xavierlepretre

Reply:
@xavierlepretre do we have a section inside our content which explains how to connect to the testnet? We need to either replace or remove the link

Reply:
We don't have any AFAIK. I wonder if this should be the object of a new section in 3-running-a-chain.